torch/csrc/jit/python/script_init.cpp

```diff
@@ -1681,6 +1704,43 @@ void initJitScriptBindings(PyObject* module) {
               ++defs_it;
               ++defaults_it;
             }
+          })
+      .def(
+          "_create_hooks",
+          [](std::shared_ptr<ConcreteModuleType> concreteType,
+             const std::vector<Def>& hookDefs,
+             const std::vector<ResolutionCallback>& hookRcbs,
+             const std::vector<FunctionDefaults>& hookDefaults,
```

Comment:
Does it make sense for hooks to have default values for arguments? Also, this doesn't seem to be used anywhere?

Reply:
Sorry, I forgot I had this logic here. I initially thought eager handled default arguments but it doesn't.